site/content/integrate/apps/_index.md

```diff
@@ -33,20 +33,20 @@ Read the [JavaScript start guide]({{< ref "quick-start-js" >}}) or [Go quick sta
 
 ### When would you build an app vs a custom slash command vs a webhook vs a plugin?
 
-That depends on your use case, as they each have benefits. Apps can be advertised, shared, and installed by others via the Marketplace. Comparatively, a custom slash command requires some configuration and is less interactive.
+That depends on your use case, as they each have benefits.
 
-The built-in incoming webhook is rigid and requires specific fields to be posted to it's URL, but some external systems don't allow the customization of the fields included in the webhook payload. Compared to built-in webhooks, An App can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
+The built-in incoming webhook is rigid and requires specific fields to be posted to its URL - but some external systems don't allow the customization of the fields included in the webhook payload. Compared to built-in webhooks, an App can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
```

Comment:
As pointed out in https://github.com/mattermost/mattermost-developer-documentation/pull/804#discussion_r622019565 I think need to emphasize more the one rice UX on webapp and _mobile_. That is something that plugins can't do.

Also, it's easier to develop an app then a plugin.

Reply:
```suggestion
The built-in incoming webhook is great for simple use cases. It requires the incoming payload to contain a valid [Post](https://pkg.go.dev/github.com/nhannv/mattermost-server/model#Post) JSON object - but some external systems don't allow the customization of the fields included in the webhook payload. Compared to built-in webhooks, an App or plugin can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
```